app/resources/views/pages/dominion/advisors/rankings.blade.php

```diff
@@ -87,7 +87,7 @@
                 </div>
                 <div class="box-body">
                     <p>The rankings advisor tells you how well all the dominions are doing in the world.</p>
-                    <p>Rankings are updated every day. Current displayed rankings are from {{ today()->diffForHumans() }}.</p>
+                    <p>Rankings are updated every day. Current displayed rankings are from {{ today() }}.</p>
```

Comment:
This still shows the wrong hour.

Reply:
This is already fixed in develop, so no worries there 😄